docs-2.0/3.ngql-guide/20.appendix/keywords-and-reserved-words.md

```diff
@@ -13,7 +13,9 @@ nebula> CREATE TAG SPACE(name string);
 Execution succeeded
 ```
 
-`TAG` is a reserved keyword. To use `TAG` as an identifier, you must quote it with a backtick. `SPACE` is a non-reserved keyword. You can use `SPACE` as an identifier without quoting it.
+`TAG` is a reserved keyword. To use `TAG` as an identifier, you must quote it with a backtick. `SPACE` is a non-reserved keyword. You can use `SPACE` as an identifier without quoting it. 
+
+> **NOTE**: There is a small pitfall when you use the non-reserved keyword. Non-quoted non-reserved keyword will be converted to **lower-case** words. Such as `SPACE` or `Space` will become `space`.
```

Comment:
Non-quoted non-reserved keyword => Unquoted and non-reserved keywords

    Non-quoted are seldom used. What about using the more popular word "Unquote" instead?

Such as `SPACE` or `Space` will become `space`. => For example, `SPACE` or `Space` will become `space`.

    While leading a complete sentence like "`SPACE` or `Space` will become `space`", "For example" would be much more appropriate.